pkg/config/agent/config.go

```diff
@@ -307,3 +309,23 @@ type PolicyBypassRule struct {
 	// The destination port of the given protocol.
 	Port int `yaml:"port,omitempty"`
 }
+
+type SecondaryNetworkConfig struct {
+	// Enable Antrea's native secondary network configuration.
+	Enable bool `yaml:"enable,omitempty"`
```

Comment:
This should be updated too.

Reply:
I meant moving it to SecondaryNetworkOVS

Reply:
Got aligned.

Reply:
Reminder.